.github/workflows/build-test.yml

```diff
@@ -35,6 +35,47 @@ jobs:
               env:
                 WHITESOURCE_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
               run: ./ci/build.sh
+            - name: Upload Build Artifact
+              uses: actions/upload-artifact@v3
+              with:
+                name: artifacts
+                path: artifacts
```

Comment:
are we really using the artifacts?

Reply:
Yes, for Windows we download the jar artifacts in the machine using the actions/download-artifact@v3

Reply:
but do we do anything with those artifacts? for what do we need them?

Reply:
We only build the module once and generate the jar. The jar files are downloaded to each machine and placed in the lib folder, which is expected to be used by the pom in the test. If the jar are not required we could removed but I am not 100% sure

Reply:
Let's test it - I think we are not using it now and building fresh jar via maven for tests

Reply:
@sfc-gh-dprzybysz Removed from actions, the current executing works with a fresh jar